cockroachcloud/cloud-sso-sql.md

```diff
@@ -0,0 +1,140 @@
+toc: true
+docs_area: manage
+---
+
+Cluster SSO allows users to access the SQL interface of a CockroachDB cluster (whether provisioned on {{ site.data.products.db }} or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of SSO identity providers, including {{ site.data.products.db }}, Google, Microsoft, GitHub, or your own self-hosted SAML or OIDC.
+
+This page describes:
+
+- [Authenticating human users](#authenticate-human-users-with-the-cloud-console) with the {{ site.data.products.db }} Console acting as identity provider (IdP) or token issuer
+- [Authenticating service accounts](#authenticate-software-users-service-accounts-with-external-idps) with an external token issuer
+- [Configuring the cluster settings](#configure-your-cluster-to-accept-your-external-identity-provider) required to authenticate service accounts
+
+Note that this regards SQL access to a specific CockroachDB Cluster, not access to a {{ site.data.products.db }} organization. For the latter, see [Single Sign-On (SSO) for CockroachDB Cloud organizations](cloud-org-sso.html).
+
+You might also be looking for [Cluster Single Sign-On (SSO) for Self-hosted CockroachDB](../v22.2/sso-sql.html).
+
+## Authenticate human users with the cloud console
+
+{{ site.data.products.db }} can issue authentication tokens for human users, which can be used for SSO using the `ccloud` CLI. {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} clusters are, by default, configured to allow Cluster SSO authentication with {{ site.data.products.db }} serving as SSO provider. No additional set-up is required.
+
+{{site.data.alerts.callout_info}}
+Note that this authentication method only works for human users, since only humans may have {{ site.data.products.db }} Console identities.
+
+Software users (i.e. service accounts), can authenticate using JWT tokens from your own identity provider. See [Authenticating software users (service accounts) with Cluster SSO](#authenticate-human-users-with-the-cloud-console).
+{{site.data.alerts.end}}
+
+**Learn more:**
+
+- [Single Sign-On (SSO) for CockroachDB Cloud organizations](cloud-org-sso.html#cloud-organization-sso)
+- [Configure Cloud Organization Single Sign-On (SSO)](configure-cloud-org-sso.html)
+
+**Prerequisites:**
+
+- You must have a user identity on a {{ site.data.products.db }} organization. For help setting up an organization and cluster, see: [Quickstart with CockroachDB](quickstart.html).
+- SSO must be enabled for your organization. For help configuring SSO for your {{ site.data.products.db }} organization, see: [Configure Cloud Organization Single Sign-On (SSO)](configure-cloud-org-sso.html)
+- SSO must be enabled for your particular {{ site.data.products.db }} user. Configure this at the [{{ site.data.products.db }} Console account settings page](https://cockroachlabs.cloud/account/profile).
```

Comment:
This is relevant only when use is using Basic SSO and Cloud Org SSO is not setup for the org.